### PR TITLE
Isaiah/lisa2

### PR DESCRIPTION
**Pull request by isaiahduncan**
[LISAv2] Skip NVME-MAX-DISK-VALIDATION test strictly for centos 7.5 and below

[DPDK] added /usr/local/lib64/pkgconfig/ to PKG_CONFIG_PATH
Problem: pkgconfig on RHEL 7 was not checking /usr/local/lib64/pkgconfig/ path
Fix: added /usr/local/lib64/pkgconfig/ to PKG_CONFIG_PATH

### CODE REVIEW

**Comment by isaiahduncan** [comment]
@johnsongeorge-w

**Comment by LiliDeng** [comment]
@isaiahduncan Why we need change NVME case for centos?

**Comment by isaiahduncan** [comment]
@LiliDeng We are skipping this specific test for only certain distros because there is a known issue with this test in certain Cent OS distros (7.5 and below) and we don't want the issue to cause unnecessary noise. 
More detail here: https://microsoft.visualstudio.com/OS/_workitems/edit/33193680

File: Testscripts/Linux/run_dpdk_ring_latency.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@ HOMEDIR=$(pwd)
 export RTE_SDK="${HOMEDIR}/dpdk"
 export RTE_TARGET="x86_64-native-linuxapp-gcc"
 export DPDK_RING_PING_PATH="${HOMEDIR}/dpdk-ring-ping"
+export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/local/lib64/pkgconfig/"
```
Comment (johnsongeorge-w) [inline Testscripts/Linux/run_dpdk_ring_latency.sh:11]: Why this change is required?

Comment (isaiahduncan) [reply Testscripts/Linux/run_dpdk_ring_latency.sh:11]: this is actually a change that matthew and I had made earlier. It is a fix to a dpdk test

Comment (isaiahduncan) [reply Testscripts/Linux/run_dpdk_ring_latency.sh:11]: The problem that prompted this change was that pkgconfig on RHEL 7 was not checking /usr/local/lib64/pkgconfig/ path
The fix for this problem was to add /usr/local/lib64/pkgconfig/ to PKG_CONFIG_PATH

Comment (isaiahduncan) [reply Testscripts/Linux/run_dpdk_ring_latency.sh:11]: @johnsongeorge-w I can remove the commit with the "export" change if you would like so that only my changes to the NVME test case are merged.

**Review by johnsongeorge-w — APPROVED** [review]
